CONTRIBUTING.md

```diff
@@ -0,0 +1,97 @@
+# Contributing to solids4foam
+
+Thank you for considering contributing to **solids4foam**!
+Thanks for helping create a world where fluid and structure are solved using your favorite CFD code.
+
+## How Can I Contribute?
+
+### 1. Reporting Bugs
+
+If you encounter a bug, please open an [GitHub issue](https://github.com/solids4foam/solids4foam/issues) and include:
+
+- A concise and descriptive title.
+
+- Steps to reproduce the issue along with and explanation of the expected behavior.
+
+- Relevant details, such as screenshots, logs, or error messages and an environment description in the following form:
+
+    ```plaintext
+    - OS version: Ubuntu 22.04.5 LTS
+    - OpenFOAM version: openfoam2312
+    - solids4foam version: 2.1
+    - Docker image (if applicable)
+    ```
+
+### 2. Suggesting Features
+
+We welcome ideas for new features! To suggest a feature:
+
+- Open a [GitHub issue](https://github.com/solids4foam/solids4foam/issues) with title: **Feature request: feature title**.
+- Provide a detailed description of the proposed feature.
+- Explain its potential benefits for the solids4foam toolbox.
+
+### 3. Submitting Code Changes
+
+#### Step 1: Fork and Clone
+
+1. Fork the repository by clicking the "Fork" button on GitHub.
+
+2. Clone your fork locally:
+
+    ```bash
+    git clone https://github.com/your-username/solids4foam.git
+    cd solids4foam
+    ```
+
+#### Step 2: Create a Branch
+
+Create a new branch for your changes:
+
+```bash
+git checkout -b branch_name
+```
+
+Branch naming conventions:
+
+- **New features**: Start with `feature-name` (e.g., `feature-newSolver`).
+- **Bug fixes**: Start with `fix-name` (e.g., `fix-boundaryConditions`).
+- **Code refactoring**: Start with `refactor-name` (e.g., `refactor-meshHandling`).
+- **Documentation changes**: Start with `docs-name` (e.g., `docs-update-readme` or `docs-fix-typos`).
```

Comment:
Since you are suggesting a fork-based workflow, does the naming convention matter?
That would maybe be more important in the PR description (and the maintainers could use labels to mark these PRs).

Reply:
Resolved in d7608d73